src/test/kotlin/nl/info/client/pabc/PabcClientServiceTest.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.info.client.pabc
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.collections.shouldContainExactlyInAnyOrder
+import io.kotest.matchers.shouldBe
+import io.mockk.checkUnnecessaryStub
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.slot
+import nl.info.client.pabc.model.createApplicationRolesResponse
+import nl.info.client.pabc.model.generated.GetApplicationRolesRequest
+
+class PabcClientServiceTest : BehaviorSpec({
+
+    val pabcClient = mockk<PabcClient>()
+
+    val roles = listOf(
```

Comment:
It is better to use fictional names for data in test instead of real names. This is now quite confusing I think because it now appears that we are tested functionality for these specific role names and that is not true at all.